### PR TITLE
Added plugin-ideal-image

### PR DESCRIPTION
**Pull request by alexvarko**

### CODE REVIEW

File: src/css/custom.css [hunk 1]
```diff
@@ -560,6 +560,10 @@ span.DocSearch-Button-Placeholder {
   background: none !important;
 }
 
+.idealImage{
+  margin-bottom: 20px;
+}
```
Comment (ardier16) [inline src/css/custom.css:565]:
```suggestion
.idealImage {
  margin-bottom: var(--ifm-paragraph-margin-bottom);
}
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -13,6 +13,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   - Added the "interoperability with external credentials" page;
 - Use-cases:
   - Added the "Age restricted content" page;
+  - Added plugin-ideal-image for image optimization
```
Comment (ardier16) [inline CHANGELOG.md:16]: change to:
```suggestion
  - Used `@docusaurus/plugin-ideal-image` for image optimization
```

and move under `### Changed` section

**Review by ardier16 — APPROVED** [review]

File: docusaurus.config.js [hunk 3]
```diff
@@ -203,5 +203,15 @@ module.exports = {
     }),
 
   /** Plugins https://docusaurus.io/docs/using-plugins */
-  plugins: [],
+  plugins: [
+    [
+      require.resolve('@docusaurus/plugin-ideal-image'),
+      {
+        max: 1920, 
+        min: 128,
```
Comment (ihordiachenko) [inline docusaurus.config.js:211]: 128px is unreasonable for charts. Let's set it to 720 or something like this

File: docusaurus.config.js [hunk 4]
```diff
@@ -203,5 +203,15 @@ module.exports = {
...
 
   /** Plugins https://docusaurus.io/docs/using-plugins */
-  plugins: [],
+  plugins: [
+    [
+      require.resolve('@docusaurus/plugin-ideal-image'),
+      {
+        max: 1920, 
+        min: 128, 
+        steps: 5,
```
Comment (ihordiachenko) [inline docusaurus.config.js:212]: Let's set it to 3 to not slow down the build too much

**Pull request merged**